Watchman.Discord/Framework/Server.cs

```diff
@@ -24,12 +24,27 @@ public static void Initialize(DiscordSocketClient client, string mongoDbConnecti
             Initialized = true;
         }
 
+        public static SocketChannel GetChannel(ulong channelId)
```

Comment:
To bym zmienił na internal
a framework bym wrzucił do osobnej DLL 
Devscord.Framework
która by siedziała w osobnym repozytorium i najlepiej jeszcze na nugecie 

nie chcemy uzależniać reszty aplikacji od Discord.NET, żeby w przyszłości mieć możliwość łatwej podmiany tej biblioteki na coś innego -> gdyby pojawiła się lepsza
lub na podmiane własną biblioteką, gdybyśmy chcieli mieć większą kontrolę nad tym, w jaki sposób komunikujemy się z discordem